packages/manager/modules/adp/src/deploy/deploy.html

```diff
@@ -0,0 +1,71 @@
+<section>
+    <!-- page header -->
+    <oui-page-header
+        data-heading="{{ ::'adp_deploy_title' | translate }}"
+        data-description="{{ ::'adp_deploy_description' | translate }}">
+    </oui-page-header>
+    <!-- page content -->
+    <div class="mx-auto" style="max-width: 1200px;">
+        <!-- error messages -->
+        <div class="mt-1 mt-sm-2" data-ng-if="$ctrl.messages.length > 0">
+            <oui-message
+                data-ng-repeat="message in $ctrl.messages track by $index"
+                data-ng-if="message.origin === 'adp.deploy'"
+                data-type="{{ message.type }}">
+                <span data-ng-bind="message.text"></span>
+            </oui-message>
+        </div>
+        <!-- deploy stepper -->
+        <div class="mt-1 mt-sm-2">
+            <oui-stepper>
+                <!-- general information -->
+                <oui-step-form data-header="{{ ::'adp_deploy_general_configuration_title' | translate }}"
+                               data-loading="$ctrl.capabilities.loading"
+                               data-on-submit="$ctrl.submitGeneralInformation()"
+                               data-valid="$ctrl.isGeneralConfigComplete()">
+                    <div data-ng-include="'adp/deploy/general.html'"></div>
```

Comment:
IMHO, we should avoid `ng-include/templateCache` and create reusable `component` instead.

Reply:
Since this cannot be re-used anywhere else, we used `data-ng-include` just to make the code more organised and readable